runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -106,13 +107,37 @@ private void init()
 		add(food(10), EEL_SUSHI);
 
 		// Alcoholic Beverages
+		// Many of these retrieved from https://oldschool.runescape.wiki/w/User:Choppe/Sandbox2
 		add(combo(food(11), dec(ATTACK, 2)), JUG_OF_WINE);
 		add(combo(food(14), dec(ATTACK, 3)), BOTTLE_OF_WINE);
 		add(combo(2, food(5), boost(STRENGTH, 6), heal(ATTACK, -4)), PREMADE_WIZ_BLZD, WIZARD_BLIZZARD);
 		add(combo(2, food(5), boost(STRENGTH, 4), heal(ATTACK, -3)), PREMADE_SGG, SHORT_GREEN_GUY);
 		add(combo(2, food(5), boost(STRENGTH, 7), heal(ATTACK, -4)), PREMADE_DR_DRAGON, DRUNK_DRAGON);
 		add(combo(2, food(5), boost(STRENGTH, 7), heal(ATTACK, -4)), PREMADE_CHOC_SDY, CHOC_SATURDAY);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.05, 1)), new BoostedStatBoost(ATTACK, false, perc(0.02, -3))), BRANDY);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.05, 1)), new BoostedStatBoost(ATTACK, false, perc(0.02, -3))), GIN);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.04, 1)), new BoostedStatBoost(ATTACK, false, perc(0.05, -3))), GROG);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.05, 1)), new BoostedStatBoost(ATTACK, false, perc(0.02, -3))), VODKA);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.05, 1)), new BoostedStatBoost(ATTACK, false, perc(0.02, -3))), WHISKY);
```

Comment:
Would it be better to group some of these items together in the same line? I see it's done above with regular food, but it may not be a big deal

Reply:
I have combined the hard liquors into one hangover inducing line

Reply:
Cheers!